recipes/emperor/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "1.0.0beta4" %}
```

Comment:
Thanks @jakirkham, my opinion has already been voiced over at https://github.com/conda-forge/conda-forge.github.io/issues/147 i.e. add an equivalent of `pip install --pre` in `conda`.

> Would it be alright to do an official release to get this out there?

The 1.0.0 release of emperor is not quite ready, there are still a few things that can and will be improved, so I'm afraid we can't really make the switch from `1.0.0beta*` to `1.0.0` :shipit: . However this version is useful as is, and is the version we have released over at [biocore](https://anaconda.org/biocore/emperor) (this is the version we recommend our users install for the time being, see this [link](https://github.com/biocore/emperor/blob/new-api/INSTALL.md), as a developer keeping everything in conda-forge would make the release process a lot simpler).

I also understand that, pre-releases are not quite supported by this infrastructure, so I'm not sure what is the best path moving forward.